euth/ideas/models.py

```diff
@@ -22,6 +25,10 @@ def get_absolute_url(self):
         from django.core.urlresolvers import reverse
         return reverse('idea-detail', args=[str(self.slug)])
 
+    def clean(self):
```

Comment:
I used this code as example: https://docs.djangoproject.com/es/1.9/ref/models/instances/#django.db.models.Model.clean